registration-processor/pre-processor/registration-processor-operator-validator-stage/src/test/java/io/mosip/registration/processor/operator/OperatorValidatorTest.java

```diff
@@ -189,9 +187,6 @@
 	@Before
 	public void setUp() throws Exception {
 		classLoader = getClass().getClassLoader();
-		File childFile = new File(classLoader.getResource("Child_ID.json").getFile());
```

Comment:
Why these lines are removed? Is the unit test cases running fine in cmd, operator, supervisor, introducer validator stages after these changes ?